plugins/arElasticSearchPlugin/lib/arElasticSearchMapping.class.php

```diff
@@ -482,4 +484,27 @@ protected function getNestedI18nObjects($languages, $nestedI18nFields)
 
     return $mapping;
   }
+
+  /**
+   * Add "sort" ES fields
+   *
+   * Add i18n "sort" field that is not tokenized, lowercase, has whitespace
+   * trimmed, punctation removed, and is ASCII folded for more natural sorting
+   */
+  protected static function addSortFields(&$nestedI18nFields, $typeProperties)
```

Comment:
Why static?

Reply:
@jraddaoui just because neither of the input variables are stored in the object context.  I lean towards static methods by default, unless there is a need to get or set the object state.

Reply:
Looks like I might be "doing it wrong" though 😛 
https://stackoverflow.com/questions/33705976/when-should-i-use-static-methods